psydac/api/ast/expr.py

```diff
@@ -227,24 +256,21 @@ def _initialize(self):
         # ...
         atoms_types = (_partial_derivatives,
                        _logical_partial_derivatives,
-                       ScalarField,
-                       VectorField, IndexedVectorField,
+                       ScalarTestFunction,
+                       VectorTestFunction, IndexedTestTrial,
```

Comment:
Do I understand correctly that we are not using `ScalarField` and `VectorField` in Psydac anymore?

Maybe it's time to fix SymPDE's issue [#50](https://github.com/pyccel/sympde/issues/50), what do you think?

Reply:
yes we are not using it, and yes we can remove it from Sympde but it will need some time to remove it from all the files

Reply:
OK, I am going to pin that issue in order to give it some priority.